staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -178,6 +178,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		endpointsLister: s.endpointsLister,
 	}
 	s.GenericAPIServer.HandlerContainer.Handle("/apis", apisHandler)
+	s.GenericAPIServer.HandlerContainer.Handle("/apis/", apisHandler)
```

Comment:
> Even subpaths that are not handled by specifically registered handlers?

We will use FallThroughHandler for URLs that are not /apis or /apis/
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis.go#L83